src/xmpp/iq.c

```diff
@@ -1612,35 +1612,54 @@ _version_get_handler(xmpp_stanza_t* const stanza)
     xmpp_ctx_t* const ctx = connection_get_ctx();
     const char* id = xmpp_stanza_get_id(stanza);
     const char* from = xmpp_stanza_get_from(stanza);
+    ProfAccount* account = accounts_get_account(session_get_account_name());
+    GString* client = account->client != NULL ? g_string_new(account->client) : NULL;
+    bool is_custom_client = client != NULL;
+    gchar* custom_version_str = NULL;
+    if (is_custom_client) {
+        custom_version_str = strstr(client->str, " ");
+        if (custom_version_str != NULL) {
+            *custom_version_str = '\0'; // Split string on name and version
```

Comment:
This part is changing the string. If it will change account->client, it can create issues. Potentially this part can later be moved in account class, resulting in less computations, but it's a really minor issue as this function is being O(n) thus does not have big impact and rarely called (on request by specific users and on login).

Reply:
That's the part I missed.